samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -457,9 +457,10 @@ LTO is an advanced compilation technique that optimizes across all compiled unit
 See :ref:`cmake_options` for detailed instructions on how to add the ``FILE_SUFFIX=release`` option to your build.
 For example, when building from the command line, you can add it as follows:
 
-.. code-block:: console
+.. parsed-literal::
+   :class: highlight
 
-   west build -b <board_name> -- -DFILE_SUFFIX=release
+   west build -b *board_target* -- -DFILE_SUFFIX=release
```

Comment:
non blocking but the `<` and `>` make it obvious to a user that they need to change something more than `*` (which I assume just makes it bold?)

Reply:
It makes it italic, hence more noticeable in the doc rendering. This is the formatting we've been using for all custom parameters.